bioio_ome_zarr/tests/test_s3_read.py

```diff
@@ -8,8 +8,9 @@ def test_ome_zarr_reader() -> None:
     uri = (
         # Cannot use s3:// URL due to ome-zarr issue #369
         # "s3://allencell/aics/nuc_morph_data"
-        "https://allencell.s3.amazonaws.com/aics/nuc_morph_data"
-        "/data_for_analysis/baseline_colonies/20200323_09_small/raw.ome.zarr"
+        "https://allencell.s3.amazonaws.com/aics/nuc-morph-dataset"
+        "/hipsc_fov_nuclei_timelapse_dataset/hipsc_fov_nuclei_timelapse_data_used_for_analysis"
+        "/baseline_colonies_fov_timelapse_dataset/20200323_09_small/raw.ome.zarr"
```

Comment:
might there be any infra concerns about egress, or monitoring statistics, with using this data for unit tests?

Reply:
I believe this is the open data bucket which, if there haven't been any major changes, is covered under AWS Open Data. I don't think Allen pays a dime for that bucket.

At least it exists on the open data Quilt page: https://open.quiltdata.com/b/allencell/tree/aics/nuc-morph-dataset/hipsc_fov_nuclei_timelapse_dataset/hipsc_fov_nuclei_timelapse_data_used_for_analysis/baseline_colonies_fov_timelapse_dataset/20200323_09_small/raw.ome.zarr/

Reply:
@evamaxfield is correct. It's possible that it may impact monitoring statistics though, so I'll check in with infra

Reply:
Also, I think if that is an issue, this PR is probably not the place to handle it, and that should be a new issue/PR.

Reply:
Done, no issues from infra